documentation/product/connectivity/topics/connections-overview.md

```diff
@@ -0,0 +1,67 @@
+# Connections Overview
+
+View and manage existing Connections within Hyperview, highlighting essential features like customizable data visibility, detailed editing options, and access control policies. By understanding how to leverage Connections effectively, users can enhance their data management, track changes, and maintain organized work notes for improved collaboration.
+
+1.	Click "Connections"
+
+```{image} /product/connectivity/media/connections-overview/image1.jpg
+:class: border-black
+```
+
+2.	Edit which columns of data are visible on the Connections overview page by Clicking the Columns Selector icon.
+
+```{image} /product/connectivity/media/connections-overview/image2.jpg
+:class: border-black
+```
+
+3.	Toggle which Columns to display.
+
+```{image} /product/connectivity/media/connections-overview/image3.jpg
+:class: border-black
+```
+
+4.	Expand a Connection to view additional data.
+
+```{image} /product/connectivity/media/connections-overview/image4.jpg
+:class: border-black
+```
+
+5.	Click the details button to view and edit all Connection properties and other information.
+
+```{image} /product/connectivity/media/connections-overview/image5.jpg
+:class: border-black
+```
+
+6.	Connections contain editable custom properties, the same that you would find on any other Asset in Hyperview.
+
+```{image} /product/connectivity/media/connections-overview/image6.jpg
+:class: border-black
+```
+
+7.	Documentation can also be added and linked to Connections.
```

Comment:
"Documentation" should probably be "Documents".

Reply:
Fixed

Reply:
I still see "Documentation" instead of "Documents" here.

Reply:
Fixed